modulefiles/module_gwci.hera.lua

```diff
@@ -0,0 +1,15 @@
+help([[
```

Comment:
I will ignore these `module_gwci.MACHINE.lua` files since we are not ready to do comparisons yet.

Reply:
Ok, yes, agreed. Not really this PR.